### ISSUE TITLE
Add email confirmation status to ChA & Mentor to Admin debugging/onboarding section

### ISSUE THREAD

**Issue by viviancan** [Iridescent-CM/technovation-app#5014]
We have had quite a few instances where program support does not why a profile may not be considered onboarded when all onboarding steps have been completed. This is sometimes due to the participant updating their email on the platform. When an email address is updated an automated email is sent and they must click "confirm" to confirm their updated email. This ticket will add the email confirmation status to the admin debugging/onboarding section for additional context. We also discussed removing this as a requirement from "onboarding."

**Comment by viviancan** [comment]
<img width="1175" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeVRRQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a854e47c957cd3b0933f18d022ad7463d33fc3f6/Screenshot%202024-09-12%20at%201.31.29%E2%80%AFPM.png" alt="Screenshot 2024-09-12 at 1.31.29 PM.png" />

**Comment by viviancan** [comment]
this is on qa for Chas and mentors

**Comment by dboyer** [comment]
Confirming that on QA I now see the "email confirmed" status on the participant admin page for mentors and ChAs. I do not see it on that page for students and admins.

I changed my password for a mentor account and saw that the status for "email confirmed" changed as expected. When I clicked the link in my email to confirm the email update, the status changed again back to "confirmed."

<img width="1142" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeTdaQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ebc4785314ce6b8c73f15d15bb9f644b0869c947/Screen%20Shot%202024-09-16%20at%2011.22.42%20AM.png" alt="Screen Shot 2024-09-16 at 11.22.42 AM.png" />
